storybookjs/storybook
# Add node_module symlink support

Issue:
If you have a symlink'd node_module then react-native adds that as a project root, as storybook is manually setting the project roots, storybook will need to detect the symlink and add it as a root

I've ran into this issue because we have a monorepo with symlink'd dependencies....

## What I did
Edited the storybook start command to call the react-native symlink detection script and add those to the project root

## How to test
- Move a node module to another location
- Create a symlink to that node_module
- Ensure that React Native has picked that module up



Thanks!
